esx_service/cli/vmdkops_admin_test.py

```diff
@@ -116,125 +116,128 @@ def test_policy_ls(self):
 
     def test_policy_ls_badargs(self):
         self.assert_parse_error('policy ls --name=yo')
+    
+    # NOTE: "tenant" is renamed to "vm-group", but we only change it in command line
+    # all the function name remain unchanged
 
     def test_tenant_create(self):
-        args = self.parser.parse_args('tenant create --name=tenant1 --vm-list vm1,vm2'.split())
+        args = self.parser.parse_args('vm-group create --name=vm-group1 --vm-list vm1,vm2'.split())
```

Comment:
I do not think there was a need to change the names of groups in the test. (no need to roll back though )